app/views/support_interface/application_forms/show.html.erb

```diff
@@ -39,18 +39,14 @@
 
 <%= render QualificationsComponent.new(application_form: @application_form, show_hesa_codes: true) %>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
```

Comment:
Removed this grid container, as no longer needed given each component dictates its own width.